### DISCUSSION TITLE
Translations

### DISCUSSION THREAD

**Discussion by vchlum** [vchlum/hue-lights#29] [Ideas]
Hi!

I  am considering translating this extension into my native language, which is Czech, and I like to ask whether there are people interested in translation(s) of this extension into other languages?

If yes, would you be willing to provide the translation of this extension in your favorite language, and occasionally update the translation?

The extension is ready to be translated since the beginning, but I did not consider it to be important. If you wish for the translation, please let me know.

Thank you,
V.

**Comment by slrtbtfs** [comment]
While I won't be using it myself (I've set the UI language to English anyway), I could try to provide a German translation, if there is interest in that.

**Comment by vchlum** [comment]
Thank you @slrtbtfs. I will tune up the text strings and circle back to you with the translation template. Making the Czech translation, I noticed some inconsistencies, spaces for improvements, and imperfections. Currently, it is around 100 strings to translate. I believe the German translation could be handy and appreciated by numerous users.

**Reply by vchlum** [reply]
Hi @slrtbtfs ! The template should be ready in hue-lights/po/template.pot If you could, please create de.po file (hue-lights/po/de.po) from the template.pot file. I would recommend using the tool [poedit](https://poedit.net/). It is often available in distros repos. Of course, you can create the file even manually:-) If you have any comments, I am here. Thank you.
